data/plugins/treeview.lua

```diff
@@ -437,15 +437,30 @@ menu:register(
 command.add(nil, {
   ["treeview:toggle"] = function()
     view.visible = not view.visible
-  end,
+  end})
+
 
+command.add(function() return view.hovered_item ~= nil end, {
```

Comment:
Could you explain why you changed the predicate function there. What was the problem with the previous approach ?

Reply:
The previous approach made the commands available everywhere, so if nothing was hovered in the `treeview` when the command was triggered, it would fail with `attempt to index field 'hovered_item' (a nil value)`.

Reply:
Fair but shouldn't we check that we have an "hovered item" and it is inside a treeview ? In other terms I guess we should check also that view is an instance of Treeview.